Sources/System/Internals/Constants.swift

```diff
@@ -454,9 +457,13 @@ internal var _O_WRONLY: CInt { O_WRONLY }
 internal var _O_RDWR: CInt { O_RDWR }
 
 #if !os(Windows)
+#if canImport(Musl)
+internal var _O_ACCMODE: CInt { 03|O_SEARCH }
```

Comment:
Copied verbatim from the header since ClangImport doesn't support C macros that expand to expressions.

Reply:
Yuck. The original `03` is in _octal_. It so happens that 03<sub>8</sub> and 03<sub>10</sub> have the same value, but that's pretty horrid. I think I would _prefer_ that if we're doing to write this in Swift code, we explicitly write `0o03` or `0x03` rather than just `03`, because the latter _looks like_ octal if you're used to C family languages, but in Swift it is in fact decimal with a leading zero.